components/Navbar.tsx

```diff
@@ -31,6 +31,11 @@ const Navbar = () => {
               Home
             </a>
           </Link>
+          <Link href="/blogs">
```

Comment:
Is link added for both mob and web view?

Reply:
Updated for mobile nav in latest commits